docs/sxpswap/users/trust-wallet.md

```diff
@@ -4,6 +4,34 @@ title: Swap/Migration for Trust Wallet users
 
 # Introduction
 
-!!! danger annotate "Coming soon"
+!!! danger annotate "Disclaimer!"
 
-    The implementation for using the swap bridge with the Trust Wallet is currently being implemented and will be available in early Q2.
+    Please read the information on this page carefully.
+    You need a Solar Wallet address to perform the swap from ERC20/BEP20 SXP to Mainnet SXP.
+    Visit our official website to download the latest [Solar Desktop Wallet](https://solar.org/desktop-wallet).
+
+
+## Swap procedure
+
+This guide is meant for users that have their ERC20/BEP20 SXP tokens in Trust Wallet.<br /> <br />
+Read the instructions below to swap your ERC20/BEP20 SXP tokens to Mainnet SXP coins:
+
```

Comment:
I leave this one up to you, but it may be helpful to mention — maybe in or around the opening paragraph — that this guide is done using Trust Wallet on iOS and that while other versions (Android, Chrome extension, etc.) may vary, the general steps are the same.